gcp/website/frontend3/src/templates/list.html

```diff
@@ -93,11 +93,15 @@ <h1 class="title">Vulnerabilities</h1>
             </span>
             <span role="cell" class="vuln-table-cell vuln-packages mdc-data-table__cell">
               <ul class="packages">
-                {% for package in (vulnerability.affected | list_packages) %}
+                {% set pkgs = vulnerability.affected | list_packages %}
+                {% for package in pkgs[:3] %}
                   <li>{{ package }}</li>
-                {% else %}
-                  <li>Not specified</li>
                 {% endfor %}
+                {% if pkgs|length > 3 %}
+                  <li>...</li>
```

Comment:
can we do something like `<li>... ({{ remaining_packages_count }} more)</li>` here with a slightly different font colour?